src/pages/challenge-specifics/component.js

```diff
@@ -20,19 +22,46 @@ const UserImage = ({userImg, className}) => (
   <img src={userImg} alt="User" className={className} />
 );
 
-export const ChallengeSpecifics = ({location, voteChallenge, user, clearChallenge}) => {
+export const ChallengeSpecifics = ({
+  location,
+  voteChallenge,
+  user,
+  clearChallenge,
+  fetchChallenge,
+  getMediaFiles,
+  challenge,
+  isFetching,
+}) => {
   const history = useHistory();
   const challengeId = location.state.challengeId;
```

Comment:
![image](https://user-images.githubusercontent.com/82516098/130192348-5d69f6fe-4486-4ee7-b459-31e08ad595cf.png)

Reply:
I have an older version of this branch